### PR TITLE
Specify fastai channel for conda install

### PR DESCRIPTION
**Pull request by NirantK**
I was unable to download fastcore from the main channel but the fastai channel works. So I am guessing this was the intended command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 > Python goodies to make your coding faster, easier, and more maintainable
 
 
-Python is a powerful, dynamic language. Rather than bake everything into the language, it lets the programmer customize it to make it work for them. `fastcore` uses this flexibility to add to Python features inspired by other languages we've loved, like multiple dispatch from Julia, mixins from Ruby, and currying, binding, and more from Haskell. It also adds some "missing features" and clean up some rough edges in the Python standard library, such as simplifying parallel processing, and bringing ideas from NumPy over to Python's `list` type.
+Python is a powerful, dynamic language. Rather than bake everything into the language, it lets the programmer customize it to make it work for them. `fastcore` uses this flexibility to add to Python features inspired by other languages we've loved, like multiple dispatch from Julia, mixins from Ruby, and currying, binding, and moSpecify fastai Channelre from Haskell. It also adds some "missing features" and clean up some rough edges in the Python standard library, such as simplifying parallel processing, and bringing ideas from NumPy over to Python's `list` type.
```
Comment (hamelsmu) [inline README.md:5]: sp: moSpecify fastai Channelre from Haskell

**Review by hamelsmu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NirantK) [reply README.md:5]: Hey @hamelsmu - I do not quite understand the changes made here. Can you please elaborate?

Comment (hamelsmu) [reply README.md:5]: @NirantK you made these changes, I am just commenting that the changes you are proposing do not seem to make any sense or possibly a spelling `sp` error on your part?

Comment (NirantK) [reply README.md]: Ohhh. That is what the `sp` meant. Yes. That was a typo/error on my part. Fixing it right now.

**Pull request merged**